### ISSUE TITLE
Make --print-queue output match the required input of --enqueue & control file

### ISSUE THREAD

**Issue by Jimi-James** [quodlibet/quodlibet#1979]
I may have mentioned before that I have a script to manage quodlibet's queue with Dropbox for me. Well, I've run into major trouble with that script. I've been trying to modify it to support queues that are longer than bash's maximum amount of arguments. Your first thought should be: why should the max args even be an issue when I can pipe it into something? The answer is, I can't, because --print-queue gives me crap that does NOT match the input syntax of commands like enqueue, like file:// and HTML character codes, and to fix that, I have to do this:
`python2 -c "import sys, urllib as ul; print ul.unquote_plus(sys.argv[1])" "$(quodlibet --print-queue | sed 's|file://||g')"`
Which, now that I've modified it to support long queues, has turned into:
`quodlibet --print-queue | xargs -P 6 -I{} -d '\n' python2 -c "import sys, urllib as ul; print ul.unquote_plus(sys.argv[1])" "{}" | sed 's|file://||g'`
Long story short, it refused to work without the -I{} option, which slows things way down and makes this way too unwieldy, which the -P 6 option does not do a good enough job of fixing.

If --print-queue would just have proper output--namely by feeding it through that python code I'm using IN the program itself before outputting--I could just do `quodlibet --print-queue` with no problem.

**Comment by declension** [comment]
Yes, it's a bit confused.
`--print-queue` and `--print-playlist` return URL results whereas `--print-query='blah'` (and of course `--print-playing` is human-readable information string, i.e. neither)

I think we should have a global toggle like `--format=text` or `--format=url` (plus perhaps `--format=json`...) and then this would be solved. The problem, and how we got here, is not wanting to break compatibility with existing scripts, so I propose we maintain the current defaults as just that, but any format flag will override.

**Comment by Jimi-James** [comment]
That sounds alright to me.
